### PR TITLE
chore: update codeowners and readme

### PR DESCRIPTION
**Pull request by christian-kreuzberger-dtx**
## This PR

- Adds @christian-kreuzberger-dtx as a codeowner
- Updates README installation instructions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,25 +1,12 @@
...
-    The supported default SLIs are:
-
-    - throughput
-    - error_rate
-    - response_time_p50
-    - response_time_p90
-    - response_time_p95
-
-The provided SLIs are based on the [RED metrics](https://grafana.com/files/grafanacon_eu_2018/Tom_Wilkie_GrafanaCon_EU_2018.pdf)
+1. Configuring Prometheus for monitoring services managed by Keptn,
```
Comment (thisthat) [inline README.md:7]: nit
```suggestion
1. configuring Prometheus for monitoring services managed by Keptn, 
```

**Review by thisthat — DISMISSED** [review]

File: README.md [hunk 2]
```diff
@@ -43,11 +30,80 @@ Please always double-check the version of Keptn you are using compared to the ve
...
+```
+
+### Install prometheus-service
+
+Please replace the placeholders in the commands below. Examples are provided.
+
+* `<VERSION>`: prometheus-service version, e.g., `0.7.1`
+* `<PROMETHEUS_NS>`: If prometheus is installed in the same Kubernetes cluster, the namespace needs to be provided, e.g., `monitoring`
+* `<PROMETHEUS_ENDPOINT>`: Endpoint for prometheus (primarily used for fetching metrics), e.g., `http://prometheus-server.monitoring.svc.cluster.local:80`
+* `<ALERT_MANAGER_NS>`: if prometheus alert manage ris installed in the same Kubernetes cluster, the namespace needs to be provided, e.g., `monitoring`
```
Comment (bacherfl) [inline README.md:53]: "alert manage ris" -> "alert manager is"

Comment (christian-kreuzberger-dtx) [reply README.md]: fixed

**Review by bacherfl — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -43,11 +30,80 @@ Please always double-check the version of Keptn you are using compared to the ve
...
+
+* Execute the following command to configure Prometheus and set up the rules for the *Prometheus Alerting Manager*:
+
+```bash
+keptn configure monitoring prometheus --project=sockshop --service=carts
+```
+
+### Optional: Verify Prometheus setup in your cluster
+
+* To verify that the Prometheus scrape jobs are correctly set up, you can access Prometheus by enabling port-forwarding for the prometheus-server:
```
Comment (mowies) [inline README.md:94]:
```suggestion
* To verify that the Prometheus scrape jobs are set up correctly, you can access Prometheus by enabling port-forwarding for the prometheus-server:
```

**Pull request merged**